app/Http/Controllers/Api/UserController.php

```diff
@@ -251,6 +251,13 @@ public function store(Request $request)
             'email' => ['required', 'string', 'email', 'max:64', 'unique:users'],
             'password' => ['required', 'string', 'min:8', 'max:191'],
         ]);
+        
```

Comment:
Make it impossible to create users via the API if the option is disabled on the front-end.